resources/public/api/conf/1.0/endpoints/sample_post.raml

```diff
@@ -0,0 +1,36 @@
+is:
+  - headers.acceptJson
+  - headers.contentTypeJson
+  - headers.testScenarioHeader
+  - errors.formatNino
+  - errors.formatTaxYear
+  - errors.clientOrAgentNotAuthorised
+  - errors.notFound
+displayName: Sample Submit Endpoint [test only]
+description: This endpoint allows a user to see an example submit endpoint.
+(annotations.sandboxData): !include ../scenarios/example_test_scenarios.md
+(annotations.scope): "write:self-assessment"
+securedBy: [ sec.oauth_2_0: { scopes: [ "write:self-assessment" ] } ]
+body:
+  application/json:
+    type: !include ../schemas/sample_request.json
+    examples:
+     example-1:
+      description: sample request body
+      value: !include ../examples/request_1.json
+responses:
+  201:
+    body:
+      application/json:
+        type: !include ../schemas/sample_response.json
```

Comment:
I don't think any of these need `../` - I haven't seen it in any of the endpoints we've worked on recently. Can you remove so these don't get copied onto the new endpoints?